app/router.ts

```diff
@@ -23,6 +23,7 @@ Router.map(function() {
     this.route('search');
     this.route('institutions', function() {
         this.route('dashboard', { path: '/:institution_id/dashboard' });
+        this.route('search', { path: '/:institution_id/search' });
```

Comment:
Did we want to keep the name "discover" for the new search page? I don't have strong opinions, but reusing "discover" may be easiest so existing links won't have to update to the new route